hyperspy/_signals/signal1d.py

```diff
@@ -1432,13 +1273,13 @@ def find_peaks1D_ohaver(self, xdim=None,
                          peakgroup=peakgroup,
                          subchannel=subchannel,
                          ragged=True,
-                         parallel=parallel,
-                         max_workers=max_workers,
                          inplace=False,
-                         lazy_output=False)
-        return peaks.data
+                         **kwargs)
+        peaks.metadata.add_node("Peaks")  # add information about the signal Axes
+        peaks.metadata.Peaks.signal_axes = deepcopy(self.axes_manager.signal_axes)
```

Comment:
Does it mean that the data are in "pixels"? Why not use the axes information to convert to calibration values? If in some scenarios, it is convenient to have the data in "pixel", maybe add an argument to make it optional, with the default of returning calibrated values?

Reply:
That's just how the find_peaks (2D) method works so I copied it.  We should have the option in returning real units in both cases.  In any case saving the `axes_manager` in the metadata is a good backup.

Reply:
Indeed, it would be very good to have both (1D and 2D) returning similar output!

Reply:
Yes, but I do would want to get calibrated units without a detour through the axis manager. For me that would be the actually be the expected default behavior.